jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/PathMappings.java

```diff
@@ -49,9 +48,18 @@
     private static final Logger LOG = LoggerFactory.getLogger(PathMappings.class);
     private final Set<MappedResource<E>> _mappings = new TreeSet<>(Comparator.comparing(MappedResource::getPathSpec));
 
-    private Trie<MappedResource<E>> _exactMap = new ArrayTernaryTrie<>(false);
-    private Trie<MappedResource<E>> _prefixMap = new ArrayTernaryTrie<>(false);
-    private Trie<MappedResource<E>> _suffixMap = new ArrayTernaryTrie<>(false);
+    private final Index.Mutable<MappedResource<E>> _exactMap = new Index.Builder<MappedResource<E>>()
```

Comment:
Can you use `<>`
```suggestion
    private final Index.Mutable<MappedResource<E>> _exactMap = new Index.Builder<>()
```

Reply:
Unfortunately not, `javac` cannot infer the `Index.Builder` type.